apps/gateway/.env.example

```
@@ -1,11 +1,8 @@
-STACK_SECRET_SERVER_KEY=your-stack_secret_server_key
-VITE_STACK_PROJECT_ID=your-stack_project_id
-
-# Model API Keys
-GROQ_API_KEY=your-groq_api_key
-VOYAGE_API_KEY=your-voyage_api_key
\ No newline at end of file
+# Required to work with vertex locally
+AWS_ACCESS_KEY_ID=your-aws_access_key_id
+AWS_SECRET_ACCESS_KEY=your-aws_secret_access_key
+AWS_SESSION_TOKEN=your-aws_session_token
+AWS_REGION=us-east-1
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)